global.json

```diff
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "2.1.801"
+    "version": "3.0.100"
```

Comment:
Can you add a `"rollforward"` property such as `"latestFeature"`?

Reply:
Can you file a bug to update the schema to include that property? 😸

Reply:
@sharwell I've never used VS to edit this file :D 

Looks like someone already did: https://github.com/dotnet/cli/issues/9341

Reply:
Hey, it's open-source, nice 
https://github.com/schemastore/schemastore/

Reply:
Fixed